### PR TITLE
Display the cloud provider in the header

### LINKED ISSUE #2178: Add cloud logos to header beside model switcher

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes #2178.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- deploy with a real cloud (aws, gce, azure)
- open the gui
- the logo for the cloud should appear in the header

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6153/
Test PASSed.

**Comment by bac** [comment]
QA looks great @huwshimi. Thanks.

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js [hunk 1]
```diff
@@ -47,259 +48,6 @@ YUI.add('deployment-flow', function() {
       withPlans: React.PropTypes.bool
     },
 
-    PROVIDERS: {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: I like how this isn't here any longer.

Comment (makyo) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: +1

**Comment by makyo** [comment]
Thanks Huw.  :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**